CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+- Added PKTINFO cmsg support on Android/iOS/Linux/MacOS.
+- Added V6PKTINFO cmsg support on Android/FreeBSD/iOS/Linux/MacOS.
+- ([#990](https://github.com/nix-rust/nix/pull/990))
```

Comment:
The PR doesn't get its own bullet.  These three lines should either all share a bullet, or the "Added" lines should each get their own and each have the PR link.